docs/get-started/troubleshooting.md

```diff
@@ -0,0 +1,15 @@
+# Troubleshooting
```

Comment:
This is great!. Can you please add a reference of this page at the end of `deploy-bosh-using-arm-templates.md` and `deploy-cloudfoundry.md`?